### ISSUE TITLE
Missing elements in resulting image from svg

### ISSUE THREAD

**Issue by xDeda** [lovell/sharp#2743]
Trying to convert certain svg files [(mouse.txt)](https://github.com/lovell/sharp/files/6602298/mouse.txt) but end up getting missing elements in the png:
expected: | output:
---------- | ----------
![mouse](https://user-images.githubusercontent.com/4290245/120887643-68732200-c5f4-11eb-9978-7a50c6bf3566.png) | ![output](https://user-images.githubusercontent.com/4290245/120887403-eafae200-c5f2-11eb-8f18-397ef734541f.png) 

```
const sharp = require('sharp');
sharp('mouse.svg').toFile('output.png', (err, info) => { console.log(err, info) });
```

Weirdly, it's something different missing every time.. Might spend some time trying to figure out what sharp is skipping but wondering if there's a fix to this?

**Comment by lovell** [comment]
The `rsvg-convert` command line tool produces the same output as sharp:
```sh
$ rsvg-convert mouse.svg > mouse.png
```
![](https://user-images.githubusercontent.com/210965/121488332-e48db100-c9ca-11eb-9933-476cabcccff6.png)

Please can you report this upstream at https://gitlab.gnome.org/GNOME/librsvg

**Comment by lovell** [comment]
@xDeda Were you able to report this upstream at https://gitlab.gnome.org/GNOME/librsvg ? (I had a quick look but couldn't see a related issue.)

**Comment by xDeda** [comment]
@lovell ....no, sorry, I was being lazy back then...... but I might do some tests these days and see if I can replicate the issue and then  write an issue :)

**Comment by xDeda** [comment]
@lovell I seem to just get a blank png when I run rsvg-convert and I'd love to report an issue upstream but it's so above my head trying to get and build all the software required to update to a more recent librsvg T_T you think it would be kosher to just post them my svg and ask what's up anyway?

**Comment by lovell** [comment]
A quick way to test the latest version is via an Alpine Edge container.
```
$ docker run -it --rm alpine:edge
/ # apk add rsvg-convert
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
...
OK: 34 MiB in 52 packages
/ # rsvg-convert -v
rsvg-convert version 2.55.1
```

**Comment by xDeda** [comment]
Reported upstream at https://gitlab.gnome.org/GNOME/librsvg/-/issues/912
sry for the delay

**Comment by xDeda** [comment]
Follow-up: it seems like they fixed this upstream: https://gitlab.gnome.org/federico/librsvg/-/commit/344d4c9f9d8b4efce6064d469b71266c25fd462b

**Comment by lovell** [comment]
Thanks for the update. This was included in libsrvg v2.55.90 via commit https://gitlab.gnome.org/GNOME/librsvg/-/commit/344d4c9f9d8b4efce6064d469b71266c25fd462b and will be picked up by the prebuilt binaries provided by sharp from v0.32.0 onwards.
